modules/fuse-loader/index.js

```diff
@@ -303,24 +303,30 @@ function $trigger(name, args) {
 }
 // NOTE: Should match syntheticDefaultExportPolyfill in fuse-box-responsive-api/index.js
 function syntheticDefaultExportPolyfill(input) {
-  if (
-    input === null ||
-    ['function', 'object', 'array'].indexOf(typeof input) === -1 ||
-    (input && Object.prototype.hasOwnProperty.call(input, 'default')) // use hasOwnProperty to avoid triggering usage warnings from libraries like mobx
```

Comment:
What happened to this line and the warning about mobx?

Reply:
Sorry about the missing comments, deleting those was an accident, I meant to re-add them once the fix was complete. I changed the checks back to use `Object.prototype.hasOwnProperty` but removed the `input &&` part since it is unnecessary as we return from the function if the input is either `null` or `undefined`.